src/inmanta/data/dataview.py

```diff
@@ -1287,6 +1308,7 @@ def construct_dtos(self, records: Sequence[Record]) -> Sequence[dict[str, str]]:
             model.DiscoveredResource(
                 discovered_resource_id=res["discovered_resource_id"],
                 values=json.loads(res["values"]),
+                managed_resource_uri=f"/api/v2/resource/{res['discovered_resource_id']}" if res["managed"] else None,
```

Comment:
I assume this is correct, but don't know what would be correct here.

Reply:
I'm not sure I understand what you mean ?

Reply:
I mean that I don't know what the schema is of these URI's

Reply:
We have a ticket to document / standardize this [here](https://github.com/inmanta/inmanta-core/issues/7557)